CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+0.5.1 (2022-09-20)
+------------------
+
+* Raise the minimum dependency of Zyte API's Python API to ``zyte-api>=0.4.0``.
+  This changes all the requests to Zyte API to have have ``Accept-Encoding: br``
+  and automatically decompress brotli responses.
```

Comment:
There is also https://github.com/scrapy-plugins/scrapy-zyte-api/pull/49 in the main branch

Reply:
Good catch! Added them in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/51/commits/d983a7563b55f66eebe60d3d749ac84ce8aea2e2.